pkg/virt-controller/watch/job.go

```diff
@@ -15,7 +15,7 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-controller/services"
 )
 
-func migrationJobSelector() kubeapi.ListOptions {
+func MigrationJobSelector() kubeapi.ListOptions {
```

Comment:
What's the rationale behind making this a public function? It's fine to do that, but the "panic" instruction in this code should instead return an error if this is to be a general purpose function.